internal/kafka/internal/services/clusters.go

```diff
@@ -32,7 +32,7 @@ type ClusterService interface {
 	Create(cluster *api.Cluster) (*api.Cluster, *apiErrors.ServiceError)
 	GetClusterDNS(clusterID string) (string, *apiErrors.ServiceError)
 	GetExternalID(clusterID string) (string, *apiErrors.ServiceError)
-	// ListEnterpriseClustersOfAnOrganization - returns a list of enterprise clusters (ClusterID and Status fields only) which belong to organization obtained from the context
+	// ListEnterpriseClustersOfAnOrganization - returns a list of enterprise clusters (ClusterID, AccessKafkasViaPrivateNetwork and Status fields only) which belong to organization obtained from the context
```

Comment:
The `-` should be removed to comply with godoc standards.
```suggestion
	// ListEnterpriseClustersOfAnOrganization returns a list of enterprise clusters (ClusterID, AccessKafkasViaPrivateNetwork and Status fields only) which belong to organization obtained from the context
```

Reply:
thanks